docs/user/azure/customization.md

```diff
@@ -15,6 +15,8 @@ The following options are available when using Azure:
 * `osDisk` (optional object):
     * `diskSizeGB` (optional integer): The size of the disk in gigabytes (GB).
 * `type` (optional string): The Azure instance type.
+* `zones` (optional string slice): List of Azure availability zones that can be used (eg. ["1", "2", "3"]).
```

Comment:
nit: backticks around the array: `["1", "2", "3"]`.  Also, "eg." -> "for example," or at least "e.g." ;)

Reply:
Changed it with your suggestion